activemodel/lib/active_model/type/integer.rb

```diff
@@ -48,7 +48,7 @@ class Integer < Value
 
       def initialize(**)
         super
-        @range = min_value...max_value
+        @range = min_value..max_value
```

Comment:
Some tests exercising large values such as `#test_exists_with_large_number` were failing. They were using values in the boundaries of the range, and serialization was failing due to that. E.g: 9223372036854775808 when range is -9223372036854775808..9223372036854775808.

I think it's correct to include boundaries here.

Reply:
Scratch this, I was wrong.